python/py-commonmark/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-commonmark
 version             0.9.1
-revision            1
+revision            2
```

Comment:
don't increase the `revision` when adding subports

Reply:
Done